deployment/aptos_chain.go

```diff
@@ -7,19 +7,15 @@ import (
 
 	ccip_offramp "github.com/smartcontractkit/chainlink-aptos/bindings/ccip_offramp"
 	module_offramp "github.com/smartcontractkit/chainlink-aptos/bindings/ccip_offramp/offramp"
-)
 
-// AptosChain represents an Aptos chain.
-type AptosChain struct {
-	Selector       uint64
-	Client         aptos.AptosRpcClient
-	DeployerSigner aptos.TransactionSigner
-	URL            string
+	"github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+)
 
-	Confirm func(txHash string, opts ...any) error
+type PAptosChain struct {
+	deployment.AptosChain
 }
```

Comment:
Why is this needed?

Reply:
we don't want to take product code inside of the framework, this GetOfframpDynamicConfig seems like very product specific

Reply:
I wanted to leave it in the `chainlink`

Reply:
Just a note to follow up on in another PR because I don't want to block this that this should add a comment in explaining your reasoning because it is hard to understand witout context

Reply:
reasonable, I will do it in the follow up